Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Resources/InputActionsEditor.uxml

```diff
@@ -27,6 +27,7 @@
                 <ui:VisualElement name="body">
                     <ui:ListView focusable="true" name="action-maps-list-view" />
                 </ui:VisualElement>
+                <ui:VisualElement name="action-maps-spacer-element" style="flex-direction: column; flex-grow: 1;" />
```

Comment:
I do not understand how this relates to the fix?

Reply:
Ah, is this a menu separator item? Are you forced to create these yourself with UITK?